### ISSUE TITLE
Error in repo

### ISSUE THREAD

**Issue by qarlosalberto** [blakawk/gitlab-integration#24]
Hello,

I have a repo in my own private server in remote origin: 

URL: 
```
http://carlos.da.privateserver.com/gitlab/84_a/mod/repo.git
```
I have created the token, but I have this error:

```no Gitlab project detected in gitlab/84_a/mod/repo.git```

**Comment by blakawk** [comment]
@qarlosalberto Can you please enable debug logs and provide it (as indicated in the README of the package). Also, please be more specific about your environment (like git remote show origin would be helpful).

**Comment by blakawk** [comment]
Should be fixed together with #28
